docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+Installing a Windows VM
+=======================
+
+Simple Windows install
+----------------------
+
+If you just want something simple and you can live without some features. This works for Windows XP, 7, 10 and 11, and it may work for Windows 8 and 8.1, although this has not been tested.
+
+**Works:**
+- display (1440x900 or 1280x1024 are a nice fit onto FHD hw display)
+- keyboard (incl. correct mapping), pointing device
+- network (emulated Realtek NIC)
+
+**Does not work:**
+- copy & paste (the qubes way)
+- copying files into / out of the VM (the qubes way)
+- assigning USB devices (the qubes way via the tray applet)
+- audio output and input
```

Comment:
Audio and USB will work with features enabled (stubdom-qrexec and audio-model)

Reply:
O.K.